source/Octopus.Client.Tests/Integration/OctopusClient/TimeoutTests.cs

```diff
@@ -36,5 +35,20 @@ public void ConfiguredTimeoutWorks()
             sw.Elapsed.Should().BeLessThan(TimeSpan.FromSeconds(10));
         }
 
+        [Test]
+        public void CancellationThrowsOperationCanceledException()
+        {
+            var sw = Stopwatch.StartNew();
+            var cancellationTokenSource = new CancellationTokenSource();
+
+            var getTask = AsyncClient.Get<string>("~/", cancellationTokenSource.Token);
+            cancellationTokenSource.Cancel();
+
+            var get = () => getTask;
+            get.ShouldThrow<OperationCanceledException>()
+                .Where(ex => ex.CancellationToken == cancellationTokenSource.Token);
+
+            sw.Elapsed.Should().BeLessThan(TimeSpan.FromSeconds(10));
```

Comment:
The stopwatch is not strictly necessary but it's something the above test `ConfiguredTimeoutWorks` also does, and it seems sensible to just verify that cancellation does the same.